php/class-post-type.php

```diff
@@ -360,6 +362,7 @@ public function render_data_metabox( $post ) {
 			echo '<li>';
 			echo '<details open>';
 			echo '<summary><code>' . esc_html( $setting_id ) . '</code> ';
+			echo '<a href="#" id="' . esc_attr( $setting_id ) . '" class="snapshot-toggle-setting-removal remove"></a>';
```

Comment:
Thanks, @westonruter. [This line](https://github.com/xwp/wp-customize-snapshots/pull/84/commits/35d5bca1f03a9fb5cd1aef66a355b91dd30eba33#diff-36a80da8dbba0bdbe840617270bb1182R365) in a new commit outputs the strings, and [this function](https://github.com/xwp/wp-customize-snapshots/pull/84/commits/35d5bca1f03a9fb5cd1aef66a355b91dd30eba33#diff-19c3ccdc306cf3ee7efd933155ccae35R59) toggles between the strings in JavaScript.